### ISSUE TITLE
DashboardFilters display as inline even when popover is specified

### ISSUE THREAD

**Issue by RobinShift4** [looker-open-source/components#2995]
I'm trying to have a minimal popover display for all my filters but the DashboardFilter component renders them as inline - the default is popover but it is still rendered as inline.  Even if I manually change the UI config to be popover they are rendered as inline.

**Comment by mdodgelooker** [comment]
Indeed, the `DashboardFilter` component has not implemented support for popover vs inline layout. Take a look at the `Popover` component from `@looker/components` – it may be helpful if you want to implement the popover behavior on your end.
